code_samples/sql-basic-usage.js

```diff
@@ -35,18 +35,17 @@ const { Client, SqlColumnType, HazelcastSqlException } = require('hazelcast-clie
                 'valueFormat' = 'double'
             )
         `;
-        // executions are async, await on update count to wait for execution.
-        await client.getSql().execute(createMappingQuery).getUpdateCount();
+        await client.getSql().execute(createMappingQuery);
 
         await map.put('key1', 1);
         await map.put('key2', 2);
         await map.put('key3', 3);
 
         let result;
         try {
-            result = client.getSql().execute('SELECT __key, this FROM myMap WHERE this > ?', [1]);
-            const rowMetadata = await result.getRowMetadata();
-            const columns = await rowMetadata.getColumns();
+            result = await client.getSql().execute('SELECT __key, this FROM myMap WHERE this > ?', [1]);
+            const rowMetadata = result.getRowMetadata();
```

Comment:
This is not valid after recent changes.

And, how does the code sample checker test passed? I see this error in logs 
```
2021-09-24T10:27:11.6504752Z An error occurred while running SQL: TypeError: result.getRowMetadata is not a function
```

but it didn't exit with a non-zero code. We should definitely fix that as well.. Maybe create an issue and fix it after 5.0?

Reply:
🤔 Sure let's fix let me see..

Reply:
Fixed it, by removing unnecessary try catch blocks